### PR TITLE
get highest versioned stable action - [WEB-4206]

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do? What is the motivation?

updates `workflow` script to only write to file  the highest versioned action.

motivation: https://datadoghq.atlassian.net/browse/WEB-4206 

prevew:  https://docs-staging.datadoghq.com/stefon.simmons/highest-stable-action-version/service_management/workflows/actions_catalog/#experimental
### Merge instructions

- [ ] Please merge after Corp reviews

### Additional notes

for review: preview shows actions tagged as `stability: dev` ([this line](https://github.com/DataDog/documentation/pull/20514/files#diff-5ed255a15fad115a8b5602e6a9e0e0aedc0a5cd021281e92046ff9c10868fb58R12)): Do not merge
  - the `testAction` actions [here](https://github.com/DataDog/dd-source/blob/main/domains/workflow/actionplatform/apps/wf-actions-worker/src/runner/bundles/com.datadoghq.experimental/manifest.json#L360-L396) should be filtered to only show `testAction:V2` in the ([preview](https://docs-staging.datadoghq.com/stefon.simmons/highest-stable-action-version/service_management/workflows/actions_catalog/experimental_testaction/)). `:V2` should be stripped from the action name but you can see the action content should match `:V2` content from the [manifest.json](https://github.com/DataDog/dd-source/blob/main/domains/workflow/actionplatform/apps/wf-actions-worker/src/runner/bundles/com.datadoghq.experimental/manifest.json#L360-L396) file

### CODE REVIEW

**Review by nsollecito — APPROVED** [review]

**Review by hestonhoffman — DISMISSED** [review]
This LGTM! Let's hold off on merging for now. I'd like to touch base with the workflows team

File: local/bin/py/build/actions/workflows.py
Comment (hestonhoffman) [inline local/bin/py/build/actions/workflows.py:12]:
```suggestion
stability = 'stable'
```
Just leaving this here, so we can flip it before merging

**Review by hestonhoffman — APPROVED** [review]

File: local/bin/py/build/actions/workflows.py
Comment (StefonSimmons) [reply local/bin/py/build/actions/workflows.py]: 👍

**Review by kq668 — APPROVED** [review]

**Pull request merged**
